library/alloc/src/vec/mod.rs

```diff
@@ -296,7 +296,7 @@ mod spec_extend;
 /// on an empty Vec, it will not allocate memory. Similarly, if you store zero-sized
 /// types inside a `Vec`, it will not allocate space for them. *Note that in this case
 /// the `Vec` might not report a [`capacity`] of 0*. `Vec` will allocate if and only
-/// if <code>[mem::size_of::\<T>]\() * [capacity]\() > 0</code>. In general, `Vec`'s allocation
+/// if [`mem::size_of`]`::<T>() * `[`capacity`]`() > 0`. In general, `Vec`'s allocation
```

Comment:
This seems to be a false-positive from the lint. If I copy the original code into a local project and build the docs there is no `<T>` tag in the output.

Reply:
Considering there were hundreds of links, I mostly looked for `<code>` tags and replaced. So a few of them just got an update because of that.

Reply:
I mean I also tested the lint locally and it warned on this:

```
warning: unclosed HTML tag `T`
 --> src/lib.rs:4:30
  |
4 | /// if <code>[mem::size_of::\<T>]\() * [capacity]\() > 0</code>.
  |                              ^^^
```

Reply:
Oh indeed. Interesting. However, this is apparently the expected behaviour since we even enforce it through a test: https://github.com/rust-lang/rust/blob/master/src/test/rustdoc-ui/intra-doc/malformed-generics.rs

Reply:
Hopefully I will be able to remove this whole commit.